extensions/radxa-aic8800.sh

```diff
@@ -10,7 +10,7 @@ function extension_finish_config__install_kernel_headers_for_aic8800_dkms() {
 
 function post_install_kernel_debs__install_aic8800_dkms_package() {
 
-	if linux-version compare "${KERNEL_MAJOR_MINOR}" ge 6.17; then
+	if linux-version compare "${KERNEL_MAJOR_MINOR}" ge 6.18; then
```

Comment:
Would you mind splitting this into its own PR and send ahead? Modifying this extension should be its own commit IMHO.

Reply:
sure